package.json

```diff
@@ -1,113 +1,151 @@
-    "docs": "cd docs && gitbook serve",
-    "unit": "karma start test/unit/karma.conf.js",
-    "e2e": "node test/e2e/runner.js",
-    "pretest": "npm run lint",
-    "test": "npm run unit",
-    "ci": "karma start test/unit/karma.conf.js --auto-watch --no-single-run --reporters=dots --port=9877",
-    "version": "bumped release"
-  },
-  "keywords": [
-    "vue",
-    "vuejs",
-    "form",
-    "generator",
-    "schema",
-    "json"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/icebob/vue-form-generator.git"
-  },
-  "files": [
-    "dist/vfg.js",
-    "dist/vfg.css",
-    "dist/vfg-core.js",
-    "dist/vfg-core.css",
-    "src"
-  ],
-  "author": "Icebob",
-  "license": "MIT",
-  "devDependencies": {
-    "babel-core": "6.23.1",
-    "babel-loader": "6.3.1",
-    "babel-plugin-lodash": "3.2.11",
-    "babel-plugin-transform-runtime": "6.23.0",
-    "babel-preset-es2015": "6.22.0",
-    "babel-preset-stage-2": "6.22.0",
-    "babel-runtime": "6.23.0",
-    "bumped": "0.10.9",
-    "bumped-terminal": "0.7.3",
-    "chai": "3.5.0",
-    "conventional-changelog-cli": "1.2.0",
-    "coveralls": "2.11.16",
-    "cross-env": "3.1.4",
-    "css-loader": "0.26.1",
-    "eslint": "3.15.0",
-    "eslint-friendly-formatter": "2.0.7",
-    "eslint-loader": "1.6.1",
-    "eslint-plugin-html": "2.0.0",
-    "eslint-plugin-vue": "1.0.0",
-    "extract-text-webpack-plugin": "1.0.1",
-    "fakerator": "0.3.0",
-    "fecha": "2.3.2",
-    "inject-loader": "2.0.1",
-    "isparta-loader": "2.0.0",
-    "karma": "1.4.1",
-    "karma-chai": "0.1.0",
-    "karma-chrome-launcher": "1.0.1",
-    "karma-coverage": "1.1.1",
-    "karma-coveralls": "1.1.2",
-    "karma-mocha": "1.3.0",
-    "karma-phantomjs-launcher": "1.0.2",
-    "karma-sinon-chai": "1.2.4",
-    "karma-sourcemap-loader": "0.3.7",
-    "karma-spec-reporter": "0.0.26",
-    "karma-webpack": "1.8.0",
-    "lodash": "4.17.4",
-    "lolex": "1.5.2",
-    "mocha": "2.5.3",
-    "mocha-generators": "1.2.0",
-    "mocha-loader": "0.7.1",
-    "node-sass": "4.7.2",
-    "phantomjs-prebuilt": "2.1.14",
-    "pug": "2.0.0-beta11",
-    "pug-loader": "2.3.0",
-    "sass-loader": "3.2.0",
-    "sinon": "1.17.7",
-    "sinon-chai": "2.8.0",
-    "stats-webpack-plugin": "0.4.3",
-    "style-loader": "0.13.1",
-    "vue": "2.1.10",
-    "vue-hot-reload-api": "2.0.8",
-    "vue-html-loader": "1.2.3",
-    "vue-loader": "10.1.2",
-    "vue-multiselect": "2.0.0-beta.13",
-    "vue-style-loader": "1.0.0",
-    "vue-template-compiler": "2.1.10",
-    "webpack": "1.14.0",
-    "webpack-dev-middleware": "1.10.0",
-    "webpack-dev-server": "1.16.2",
-    "webpack-merge": "0.14.1"
-  },
-  "dependencies": {},
-  "publishConfig": {
-    "access": "public"
-  }
+	"name": "vue-form-generator",
+	"version": "3.0.0-dev",
+	"description": "A schema-based form generator component for Vue.js",
+	"homepage": "https://github.com/icebob/vue-form-generator",
```

Comment:
https://github.com/vue-generators/vue-form-generator/

Reply:
Yes, sorry, I'm going to fix that